package.json

```diff
@@ -7,7 +7,8 @@
     "build": "npm run lint && webpack --config ./webpack.config.prod.js -p",
     "develop": "webpack-dev-server --open --config ./webpack.config.develop.js",
     "lint": "eslint *.js",
-    "serve": "npm run build && http-server ./build -o -p 9000"
+    "serve": "npm run build && http-server ./build -o -p 9000",
+    "test": "jest --coverage"
```

Comment:
I wonder if it makes sense to only generate coverage when needed, as opposed to every time.  maybe a matter of perceived performance.  let's chat about approaches here.